websites/S/Stake.us/presence.ts

```diff
@@ -0,0 +1,73 @@
+const presence = new Presence({
+		clientId: "1231098376741982349",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const [showBal, showTimestamp, showCurrentGame] = await Promise.all([
+			presence.getSetting<boolean>("showBal"),
+			presence.getSetting<boolean>("showTimestamp"),
+			presence.getSetting<boolean>("showCurrentGame"),
+		]),
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/9jDaQ1b.png",
+		},
+		{ pathname, search } = window.location;
```

Comment:
For consistency across presences, use `document.location`
```suggestion
		{ pathname, search } = document.location;
```